src/exact.jl

```diff
@@ -68,26 +68,27 @@ function isdominated(A, b, islb, isfun, k, solver, lb, ub, epsilon)
         if ix != k
             ic += 1
             δb = b[k] - b[ix]
-            @inbounds h[ic] = ((isfun&islb) || (~isfun&~islb)) ? δb : -δb
+            @inbounds h[ic] = ((isfun & islb) || (~isfun & ~islb)) ? δb : -δb
             @inbounds H[ic, 1] = (islb) ? 1 : -1
 
             for jx in 1:nx
                 dl = A[ix, jx] - λk[jx]
-                @inbounds H[ic, jx+1] = (islb) ? dl : -dl
+                @inbounds H[ic, jx + 1] = (islb) ? dl : -dl
             end
         end
     end
 
     # update lower and upper bound if Vector
     lbx = isa(lb, Vector) ? vcat(-Inf, lb) : lb
-    ubx = isa(ub, Vector) ? vcat( Inf, ub) : ub
+    ubx = isa(ub, Vector) ? vcat(Inf, ub) : ub
 
-    # solve the LP with MathProgBase
-    res = linprog(c, H, -Inf, h, lbx, ubx, solver)
-    if res.status == :Optimal
-        res = res.objval
-        return (islb) ? -res < epsilon : res > -epsilon
-    else
-        return false
-    end
+    # solve the LP with JuMP
+    model = Model(optimizer_constructor)
+    z = @variable(model, [1:nx+1], lower_bound=lbx, upper_bound=ubx)
```

Comment:
Giving infinite bounds is not correctly handled by some solvers.
Creates free variables with `@variable(model, [1:nx+1])` and then do
```julia
if ub isa Vector
    set_lower_bound.(z, lb)
end
```
Same for the upper bound

Reply:
Done!